src/lazy/text/buffer.rs

```diff
@@ -499,16 +522,10 @@ impl<'data> TextBufferView<'data> {
     /// range of input bytes where the field name is found, and the value.
     pub fn match_struct_field_name_and_e_expression_1_1(
         self,
-    ) -> IonParseResult<
-        'data,
-        (
-            (MatchedFieldName, Range<usize>),
-            RawTextEExpression_1_1<'data>,
-        ),
-    > {
+    ) -> IonParseResult<'top, (MatchedFieldName, RawTextEExpression_1_1<'top>)> {
```

Comment:
🗺️ This is the return type simplification. `MatchedFieldName` now contains the equivalent of a `Range<usize>`, allowing this to be reduced.